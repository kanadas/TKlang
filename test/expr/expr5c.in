(\a b -> if a + b == /*tralalalaal*/0 then true else a + b) 0 'a'
