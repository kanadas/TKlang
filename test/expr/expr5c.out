expr/expr5c.in

Type mismatch: bool and int in expression:
if a + b == 0 then true else a + b
